pkg/restapi/internal/common/did/common.go

```diff
@@ -363,6 +364,16 @@ func (o *CommonDID) createPublicKeys(keyType, signatureType string) (*did.Doc,
 		return didDoc, pks, key3ID, nil
 	}
 
+	if keyType == crypto.Ed25519KeyType &&
+		crypto.JSONWebKey2020 == signatureKeyTypeMap[signatureType] {
+		return didDoc, pks, key2ID, nil
```

Comment:
shouldn't you delete `doc.JWSVerificationKey2020` conditions above as well? if we don't want them, they should be removed

Reply:
Done